### PR TITLE
HOMS-234 Speed up OrderController test

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ v1.5.0 [unreleased]
 -   [#161](https://github.com/latera/homs/pull/161), [#6](https://github.com/latera/activiti-homs-docker/pull/6) Use activiti credentials from environment variables.
 -   [#151](https://github.com/latera/homs/pull/151) Add current version to docker container.
 -   [#151](https://github.com/latera/homs/pull/152) Remove libqtwebkit-dev.
+-   [#180](https://github.com/latera/homs/pull/180) Reduce test runtime
```
Comment (sugarfree1) [inline CHANGELOG.md]: Dot at the end of file.

**Comment by mxsmirnoff** [comment]
Updated

**Comment by sugarfree1** [comment]
```
build	23-Oct-2018 00:02:21	Top 10 slowest examples (319.22 seconds, 36.6% of total time):
build	23-Oct-2018 00:02:21	  OrdersController Orders create failure
build	23-Oct-2018 00:02:21	    109.91 seconds ./spec/controllers/orders_controller_spec.rb:20
```

**Comment by mxsmirnoff** [comment]
Updated

**Pull request closed without merging**